### ISSUE TITLE
Not working with jupyterlab>=3

### ISSUE THREAD

**Issue by vuillaut** [nicolaschotard/stackyter#15]
I can't fire a remote `jupyterlab>=3.0` while it's perfectly working with `jupyterlab==2`
Anyone else experiencing this issue?

Here is the log:

```
[I 2021-10-22 18:24:37.519 LabApp] JupyterLab application directory is /fefs/aswg/workspace/thomas.vuillaume/miniconda3/envs/jlab/share/jupyter/lab
[I 2021-10-22 18:24:37.526 ServerApp] jupyterlab | extension was successfully loaded.
[I 2021-10-22 18:24:37.527 ServerApp] Serving notebooks from local directory: /fefs
[I 2021-10-22 18:24:37.527 ServerApp] Jupyter Server 1.11.1 is running at:
[I 2021-10-22 18:24:37.527 ServerApp] http://127.0.0.1:61802/lab?\
[I 2021-10-22 18:24:37.527 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2021-10-22 18:24:37.535 ServerApp]

    To access the server, open this file in a browser:
        file:///fefs/home/thomas.vuillaume/.local/share/jupyter/runtime/jpserver-4344-open.html
    Or copy and paste one of these URLs:
        http://127.0.0.1:61802/lab?token=ff939bdaf0fbb410a614f54645746606fe617817fbd3a926
     or http://127.0.0.1:61802/lab?token=ff939bdaf0fbb410a614f54645746606fe617817fbd3a926
waiting...
waiting...
waiting...
waiting...
waiting...
waiting...
waiting...
waiting...
waiting...
waiting...
(jlab) [thomas.vuillaume@cp02 fefs]$         if [[ -z ${servers} ]]; then
>            echo 'could not determine the URL of the Jupyter server'
>            kill -TERM ${jupyter_pid} &> /dev/null
>            exit 1
>         fi
could not determine the URL of the Jupyter server
déconnexion
[C 2021-10-22 18:25:03.102 ServerApp] received signal 15, stopping
[I 2021-10-22 18:25:03.108 ServerApp] Shutting down 2 extensions
[I 2021-10-22 18:25:03.109 ServerApp] Shutting down 0 kernels
[I 2021-10-22 18:25:03.109 ServerApp] Shutting down 0 terminals
```

**Comment by HealthyPear** [comment]
I seem to have this issue and I have installed v0.22 which was released on Oct 15th, 2021 after the merge of #14, so I assume that version should contain the fix, but I get exactly the same output.

On my remote machine I am using jupyterlab v3.6.2

**Comment by HealthyPear** [comment]
My Jupyter setup is the following,

```
IPython          : 8.11.0
ipykernel        : 6.22.0
ipywidgets       : not installed
jupyter_client   : 8.1.0
jupyter_core     : 5.3.0
jupyter_server   : 2.5.0
jupyterlab       : 3.6.2
nbclient         : 0.7.2
nbconvert        : 7.2.9
nbformat         : 5.8.0
notebook         : 6.5.3
qtconsole        : not installed
traitlets        : 5.9.0
```
